docs/getting-started/features.md

```diff
@@ -2,206 +2,230 @@
+## Kubernetes
 
-1. Click the **+** button in the top-right corner.
-1. Click on the **Pull** tab.
-1. Enter the name of the image to pull.
-    > Note: By default, images are pulled from [Docker Hub] in which use the `repo/image[:tag]` format. To pull from other registries, include the hostname `registry.example.com/repo/image[:tag]`.
-1. Click **Pull**.
+### Enable Kubernetes
 
-[Docker Hub]:
-https://hub.docker.com/
+This option allows you to enable or disable Kubernetes. By disabling Kubernetes, you can run just `containerd` or `dockerd` by itself for reduced resource consumption. By default, Kubernetes is enabled.
 
-#### Building Images
+To enable/disable Kubernetes, just check/uncheck the `Enable Kubernetes` checkbox. The app would be restarted when you enable/disable Kubernetes. Disabling Kubernetes will not delete existing resources and they will be available again when you enable Kubernetes back.
 
-Use this option to build an image and add it to your virtual machine.
+### Kubernetes Version
 
-To build an image:
+This option presents a list of Kubernetes versions that your Rancher Desktop instance can use.
 
-1. Click the **+** button in the top-right corner.
-1. Click on the **Build** tab.
-1. Enter a name for the image being built. E.g., `repo/image`, `repo/image:tag`, `registry.example.com/repo/image`, or `registry.example.com/repo/image:tag`.
-1. Click **Build**.
-1. In the file browser, select the Dockerfile to build an image with.
+When upgrading:
 
-## Troubleshooting
+- A check is performed to see if the target Kubernetes version is available locally. If not, it downloads the files.
+- Workloads are retained.
+- Images are retained.
 
-### Show Logs
+When downgrading:
 
-Use this option to open the folder containing all Rancher Desktop log files.
+- Workloads are removed.
+- Images are retained.
 
-### Factory Reset
+To switch versions:
 
-Remove the cluster and all other Rancher Desktop settings. The initial setup procedure must be done again.
+1. Click the **Kubernetes version** drop-down menu.
+1. Select the version you want to change to.
+1. On the confirmation window, click **OK** to proceed.
 
-To perform a factory reset:
+### Kubernetes Port
+
+Set the port Kubernetes is exposed on. Use this setting to avoid port collisions if multiple instances of K3s are running.
+
+### Enable Traefik
+
+This option allows you to enable or disable Traefik. By disabling Traefik, you can free up port 80 and 443 for alternate ingress configuration. By default, Traefik is enabled.
 
-1. Click **Reset**.
-1. On the confirmation window, click **OK** to proceed. Kubernetes stops and Rancher Desktop closes.
-1. Start Rancher Desktop again.
+To enable/disable Traefik, just check/uncheck the `Enable Traefik` checkbox. The app needs to be manually restarted when you enable/disable Traefik. This behavior will be changed in an upcoming release, where a prompt will let you restart the app. Disabling Traefik will not delete existing resources.
```

Comment:
We could probably remove a majority of this paragraph

```suggestion
Disabling Traefik will not delete existing resources.
```

Reply:
Feedback applied.